### PR TITLE
add new study: Singh Ramesh 2023

### PR DESCRIPTION
**Pull request by ehwenk**
add new study, Singh Ramesh 2023

there remains some concern that the leaf P values are incredibly high, but the contributor and I have both checked all conversions and it seems to be correct

### CODE REVIEW

File: config/traits.yml [hunk 1]
```diff
@@ -1450,8 +1450,10 @@ traits:
         decussate: When sequential pairs of opposite leaves attach at right angles to the pair above or below.
         distichous: When alternate or opposite leaves are arranged in two opposite rows.
         spiral: When alternate leaves are attached in a progressive spiral pattern along the stem.
-        fasciculate: A tight bundle or cluster of leaves attached to the stem in a single location, generally due to the shortening of internodes.
-        crowded: When internode length varies considerably along a stem, such that leaves appear crowded or tufted.
+        fasciculate: A tight bundle or cluster of leaves attached to the stem in a single location, generally due to the shortening of internodes. (Synonym, clustered)
+        crowded: When internode length varies considerably along a stem, such that leaves appear crowded or tufted. (Synonym, congested)
+        imbricate: XXX
```
Comment (yangsophieee) [inline config/traits.yml:1455]: Just checking if imbricate's definition was meant to be filled in or not.

File: config/traits.yml [hunk 2]
```diff
@@ -2000,7 +2017,7 @@ traits:
         widely_elliptical: 'A 2-dimensional shape that is a broad oval, with curved margins, lacking a distinct point, and a length:width ratio of approximately 6:5'
         orbicular: 'A 2-dimensional shape that is approximately circular. (synonym: circular)'
         oblate: 'A 2-dimensional shape that is a broad oval, with curved margins, lacking a distinct point, and a length:width ratio of approximately 5:6, where the petiole attaches along the broader side.'
-        lanceolate: 'A 2-dimensional shape that is narrowly egg-shaped or teardrop-shaped, with a length:width ratio between 6:1 and 3:1. The petiole attaches near the wide, curved portion of the leaf lamina.'
+        lanceolate: 'A 2-dimensional , crescenthape that is narrowly egg-shaped or teardrop-shaped, with a length:width ratio between 6:1 and 3:1. The petiole attaches near the wide, curved portion of the leaf lamina.'
```
Comment (yangsophieee) [inline config/traits.yml:2020]: Is 'crescenthape' a typo?

File: data/SinghRamesh_2019/metadata.yml [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+  Mt Windsor_6:
+    elevation (m): 1302.0
+    latitude (deg): -16.212347
+    longitude (deg): 144.999586
+contexts: .na
+traits:
+- var_in: leaf d13C
+  unit_in: '{delta}[ppth]'
+  trait_name: leaf_delta13C
+  entity_type: individual
```
Comment (yangsophieee) [inline data/SinghRamesh_2019/metadata.yml:152]: Checking that the branches sampled for each species were from different individuals (I assume that would be the case!)

File: data/SinghRamesh_2023/metadata.yml [hunk 4]
```diff
@@ -0,0 +1,459 @@
...
+    Complex (ERC), (~30 m a.s.l.) JCU, Cairns, Australia. Healthy saplings were transplanted
+    (approximately 3 months after sowing seeds) into either 13.5 L pots (Garden City
+    Pots, Model P300ST00) containing nutrient-rich soil (NR), comprising a 1:1 mixture
+    by volume of premium garden potting mix and compost (Northside Landscape Supplies
+    Pty. Ltd., Trinity Beach); or 8.5 L pots (Garden City Pots, Model P250STTL) containing
+    nutrient-poor soil (NP). The nutrientpoor soil comprised a locally collected dermasol,
+    low in organic matter and supplemented with perlite and washed river sand to improve
+    drainage (Supplementary Table 2). The two soil mixtures, nutrient-rich and nutrient-poor,
+    had similar d15N at the start of the experiment, based on measurements of subsamples;
+    these values were 5.4 and 5.3h, respectively.  ^p  The experimental pots were
```
Comment (yangsophieee) [inline data/SinghRamesh_2023/metadata.yml:59]: I don't know why but the parts-per-thousand symbol gets copied and pasted as 'h' in "5.3h".

File: config/traits.yml (hunk 2 above)
Comment (ehwenk) [reply config/traits.yml:2020]: yes

File: data/SinghRamesh_2019/metadata.yml (hunk 3 above)
Comment (ehwenk) [reply data/SinghRamesh_2019/metadata.yml:152]: yes

File: data/SinghRamesh_2023/metadata.yml [hunk 5]
```diff
@@ -0,0 +1,459 @@
...
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: Final biomass was measured from destructive harvests of three plants per
+    species by treatment combination, except for F. bourjotiana in the nutrient-rich,
+    elevated temperature treatment, where only two plants survived. Thus, a total
+    of 71 plants were harvested for final biomass. The dry mass of leaves, stems and
+    roots were measured separately and used to calculate Above Ground Biomass (AGB,
+    sum of leaf and stem dry mass) and Below Ground Biomass (BGB, total root dry mass).
+    The AGB and BGB were used to calculate the root-to-shoot ratio (R:S).
```
Comment (yangsophieee) [inline data/SinghRamesh_2023/metadata.yml:176]: I noticed the column R/S (root_shoot_ratio) isn't mapped - was this intentional?

File: data/SinghRamesh_2023/metadata.yml [hunk 6]
```diff
@@ -0,0 +1,459 @@
...
+    Stable isotope results are reported as per mil (h) deviations from the VPDB and
+    AIR reference standard scales for d13C and d15N, respectively.
+- var_in: Leaf_P_Area_recalculated_g/m2
+  unit_in: g/m2
+  trait_name: leaf_P_per_area
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: unknown
```
Comment (yangsophieee) [inline data/SinghRamesh_2023/metadata.yml:316]: How come the methods for `leaf_P_per_area`, `stem_delta13C`  and `leaf_P_per_dry_mass` are unknown when `stem_delta15N` and `leaf_N_per_are` have methods? Or is it just the phosphorus measurements that are unknown?

**Review by yangsophieee — COMMENTED** [review]
Looks great!

File: config/traits.yml (hunk 1 above)
Comment (ehwenk) [reply config/traits.yml:1455]: they were all meant to be on a different branch - I hadn't caught that I'd mixed up branches

File: data/SinghRamesh_2023/metadata.yml (hunk 4 above)
Comment (ehwenk) [reply data/SinghRamesh_2023/metadata.yml:59]: Fixed

File: data/SinghRamesh_2023/metadata.yml (hunk 5 above)
Comment (ehwenk) [reply data/SinghRamesh_2023/metadata.yml:176]: no - I've added it now

File: data/SinghRamesh_2023/metadata.yml (hunk 6 above)
Comment (ehwenk) [reply data/SinghRamesh_2023/metadata.yml:316]: just the `leaf_P` traits should have been unknown

**Review by yangsophieee — APPROVED** [review]

**Pull request merged**